packages/editor/src/components/sidebar/post-summary.js

```diff
@@ -33,30 +35,16 @@ import { store as editorStore } from '../../store';
 const PANEL_NAME = 'post-status';
 
 export default function PostSummary() {
-	const { isOpened, isRemoved, showPostContentPanels } = useSelect(
```

Comment:
We need to keep the `showPostContentPanels` check. The current logic doesn't render the `Summary` panel for the patterns when editing with a classic theme.

| trunk | this branch |
|--------|--------|
| ![CleanShot 2024-05-13 at 14 07 23](https://github.com/WordPress/gutenberg/assets/240569/18e4a825-45be-4cb3-9433-c16cad76b898) | ![CleanShot 2024-05-13 at 14 06 54](https://github.com/WordPress/gutenberg/assets/240569/d1365e0c-1430-4320-a4c3-f22a33425918) |

Reply:
It was suggested above that it's not very helpful https://github.com/WordPress/gutenberg/pull/61507#issuecomment-2102523720

Reply:
To minimize changes, I restored it but kept only the "sync status"

Reply:
I think we still want to display `Sync status` somewhere. The header has a purple accent color, but that's not very accessible.

I don't think it's a blocker here as long as we quickly ship a follow-up for it (same Gutenberg release). cc @jameskoster

Reply:
Agree sync status is useful, also (down the road) category. 

Just for reference, for patterns, this is what we're aiming at:

<img width="868" alt="Screenshot 2024-05-13 at 12 58 47" src="https://github.com/WordPress/gutenberg/assets/846565/f6e9fecb-d575-4092-8221-818829fe35a6">

Reply:
Not for patterns specifically. It's a part of https://github.com/WordPress/gutenberg/issues/59689.